src/main/java/core/basesyntax/sevice/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.sevice.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String REGEX_TO_SPLIT = ",";
+    private static final String CONTENTS = "fruit, quantity";
+
+    @Override
+    public String createReport() {
+        StringBuilder stringBuilder = new StringBuilder(CONTENTS + System.lineSeparator());
+        for (Map.Entry<String, Integer> entry : FruitStorage.fruitRepository.entrySet()) {

```

Comment:
Direct access to 'FruitStorage.fruitRepository' breaks the encapsulation principle. Consider creating a method in 'FruitDao' to access the storage data.